smartsim/_core/_cli/build.py

```diff
@@ -23,22 +23,29 @@
 
 
 def _install_torch_from_pip(versions, device="cpu", verbose=False):
+
     packages = []
     end_point = None
-    # if we are on linux cpu, use the torch without CUDA
-    if sys.platform == "linux" and device == "cpu":
-        packages.append(f"torch=={versions.TORCH}+cpu")
-        packages.append(f"torchvision=={versions.TORCHVISION}+cpu")
+
+    if sys.platform == "darwin":
+        if device == "gpu":
+            logger.warning("GPU support is not available on Mac OS X")
+        device_suffix = ""
```

Comment:
should this be `versions.TORCH_CPU_SUFFIX`?

Reply:
No, since Mac OS X does not have any suffix associated with it. I added a comment to explicitly mention this.